File: collaboration-service/src/main/java/it/domina/nimble/collaboration/exceptions/SubscriptionRequired.java
package it.domina.nimble.collaboration.exceptions;

public class SubscriptionRequired  extends Exception {

	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;

	
	public static final String ERRCODE = "SUBSCRIPTION_REQUIRED";
	
	public SubscriptionRequired() {
		super(ERRCODE);
	}
	
}

File: collaboration-client/NimbleCollaborationClient/Exceptions/SubscriptionRequired.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Nimble.Client.Exceptions
{
    public class SubscriptionRequired : Exception
    {
        
        public const String ERRCODE = "SUBSCRIPTION_REQUIRED";
	
	    public SubscriptionRequired() : base(ERRCODE) { }

    }
}
